api/src/main/resources/workbench-api.yaml

```diff
@@ -6404,6 +6409,15 @@ definitions:
       - MASTER
       - RESTRICTED
 
+  InstitutionRequirementEnum:
```

Comment:
nit: per other comment, I would probably not include "Enum" in the type name for an enum